src/common/mrob/matrix_base.hpp

```diff
@@ -68,6 +70,7 @@ typedef Eigen::Matrix<matData_t, 1,3> Mat13;
 typedef Eigen::Matrix<matData_t, 1,4> Mat14;
 typedef Eigen::Matrix<matData_t, 1,5> Mat15;
 typedef Eigen::Matrix<matData_t, 1,6> Mat16;
+typedef Eigen::Matrix<matData_t, 1,9> Mat19;
```

Comment:
actually, you can also use the form Mat<1,9> or Vec<9> but i guess this was more convenient... I